docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
     Use "dce [command] --help" for more information about a command.
     ```
 
-4. Type `dce init` to configure dce via an interactive prompt. This will generate a config file at ~/.dce.yaml
+1. Type `dce init` to generate a new config file at ~/.dce.yaml. Leave everything blank for now.
+
+## Deploying DCE
+
+1. [Download and install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+1. Choose an AWS account to be your new "DCE Master Account" and configure the AWS CLI with user credentials that have AdministratorAccess in that account.
+
+    ```
+    ➜  ~ aws configure set aws_access_key_id default_access_key
+    ➜  ~ aws configure set aws_secret_access_key default_secret_key
+    ```
+
+1. Type `dce system deploy` to deploy dce to the AWS account specified in the previous step.
+
+1. Edit your dce config file with the host and base url from the api gateway that was just deployed to your master account. This can be found in the master account under `API Gateway > (The API with "dce" in the title) > Stages > "Invoke URL: https://<host>/<baseurl>"`. Your config file should look something like this:
+
+    ```
+    api:
+      host: abcdefghij.execute-api.us-east-1.amazonaws.com
+      basepath: /api
+    region: us-east-1
+    ```
+   
+## Authenticating with the DCE System
+
+DCE uses AWS Cognito to manager authentication and authorization.
+
+1. Open the AWS Console in your DCE Master Account and Navigate to AWS Cognito by typing `Cognito` in the search bar
+
+    ![Cognito](./cognito.png)
+
+1. Select `Manage User Pools` and click on the dce user pool.
+
+    ![Manage user pools](./manageuserpools.png)
+
+1. Select `Users and groups`
 
-5. Type `dce system deploy` to deploy dce to the AWS account specied in the previous step. This will be your new "DCE Master Account"
+    ![Users and Groups](./usersandgroups.png)
+    
+1. Create a user
 
-6. Edit your dce config file with the host and base url from the api gateway that was just deployed to your master account. This can be found in the master account under `API Gateway > (The API with "dce" in the title) > Stages > "Invoke URL: https://<host>/<baseurl>"`.
+    ![Create User](./createuser.png)
 
-7. Prepare a second AWS account to be your first "DCE Child Account"
+1. Name the user and provide a temporary password. You may uncheck all of the boxes and leave the other fields blank. This user will not have admin priviliges.
+
+    ![Quick start user](./quickstartuser.png)
+    
+1. Create a second user to serve as a system admin. Follow the same steps as you did for creating the first user, but name this one something appropriate for their role as an administrator.
+   
+    ![Quick start admin](./quickstartadmin.png)
+
+1. Create a group
+
+    ![Create group](./creategroup.png)
+
+1. Users in this group will be granted admin access to DCE. The group name must contain the term `Admin`. Choose a name and click on the `Create group` button.
```

Comment:
> must contain the term `Admin`

@joshmarsh what's the story on this? Is this a mechanism of Cognito, or something we setup?

It just feels a little hacky

Reply:
This was set up a while ago after we discussed using Cognito User Pool Groups and IdP attributes as the two mechanisms by which we would assign admin/non-admin IAM roles to users.

I'm pretty sure it was intended as a "good enough" solution while we focused on getting the auth working overall. We should definitely iterate on it.